UnitTests/GitUITests/CommitInfo/CommitInfoTests.cs

```diff
@@ -74,18 +75,94 @@ public void OneTimeTearDown()
         }
 
         [Test]
-        public void GetSortedRefs_should_throw_on_git_warning()
+        public void BranchComparer([Values(null, "current")] string currentBranch)
```

Comment:
Is this a fancy replacement for `TestCase`?

Reply:
Yes, especially if you have a few arguments. It tests all combinations.